cmd/mobynit/main.go

```diff
@@ -65,9 +66,72 @@ func mountContainer(containerID, graphDriver string) string {
 	return newRootPath
 }
 
+func mountBalenaContainerLocal(layer_root, mount_dest string) string {
+
+	rawGraphDriver, err := ioutil.ReadFile(filepath.Join(layer_root,"/current/boot/storage-driver"))
+	if err != nil {
+		log.Fatal("could not get storage driver:", err)
+	}
+	graphDriver := strings.TrimSpace(string(rawGraphDriver))
+
+	current, err := os.Readlink(filepath.Join(layer_root,"/current"))
+	if err != nil {
+		log.Fatal("could not get container ID:", err)
+	}
+	containerID := filepath.Base(current)
+
+	if err := os.MkdirAll(mount_dest, os.ModePerm); err != nil {
+		log.Fatal("creating %s failed:", mount_dest, err)
+	}
+
+	ls, err := layer.NewStoreFromOptions(layer.StoreOptions{
```

Comment:
Instead of duplicating the code here you can factor our a function that only does the mounting part which will be called by `mountContainer`

Reply:
Refactored quite a bit this time round